static/templates/bot_avatar_row.handlebars

```diff
@@ -8,9 +8,23 @@
                 <button type="submit" class="btn open_edit_bot_form" data-sidebar-form="edit-bot" title="{{t 'Edit bot' }}" data-email="{{email}}">
                     <i class="icon-vector-pencil blue"></i>
                 </button>
-                <a type="submit" download="{{zuliprc}}" class="btn download_bot_zuliprc" title="{{t 'Download .zuliprc' }}" data-email="{{email}}">
+                <button class="btn open_get_zuliprc_modal" title="{{t 'Get zuliprc' }}" data-email="{{email}}">
                     <i class="icon-vector-download-alt sea-green"></i>
-                </a>
+                </button>
+                <div id="get_zuliprc_modal" class="modal hide" tabindex="-1" role="dialog"
```

Comment:
@showell strange. For example, this was part of the code I pushed this earlier this evening, funny it didn't show up in the main page.